File: Python/vehicle.py
class Vehicle:
    def __init__(self, plate_number, brand, year, color):
        self.plate_number = plate_number
        self.brand = brand
        self.year = year
        self.color = color

    def display_info(self):
        print("Plate Number:", self.plate_number)
        print("Brand:", self.brand)
        print("Year:", self.year)
        print("Color:", self.color)

File: PHP/Vehicle.php
<?php
class Vehicle {
    protected $plateNumber;
    protected $brand;
    protected $year;
    protected $color;

    public function __construct($plateNumber, $brand, $year, $color) {
        $this->plateNumber = $plateNumber;
        $this->brand = $brand;
        $this->year = $year;
        $this->color = $color;
    }

    public function displayInfo() {
        echo "Plate Number: " . $this->plateNumber . PHP_EOL;
        echo "Brand: " . $this->brand . PHP_EOL;
        echo "Year: " . $this->year . PHP_EOL;
        echo "Color: " . $this->color . PHP_EOL;
    }
}
?>
